ckanext/syndicate/fanstatic/scripts/syndicate-logs.js

```diff
@@ -0,0 +1,78 @@
+ckan.module('syndicate-log-retry', function($, _) {
+    "use strict";
+
+    function create_alert(status, text) {
+      return $('<div></div>', {
+        class: 'alert alert-' + status + ' alert-dismissable',
+        html: '<a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>' + text
+      });
+    }
+
+    return {
+        initialize: function() {
+          $.proxyAll(this, /_on/);
+          this.el.on('click', this._onClick);
+        },
+        _onClick: function(e) {
+          e.preventDefault();
+
+          var pkgId = this.options.pkgid;
+
+          $.ajax({
+            'url': '/organization/syndicate-logs/syndicate-log-retry/' + pkgId,
```

Comment:
it won't work if CKAN instance is nested into some portal and available not at the `/`. I believe that you can use `ckan.SITE_ROOT` or something like this in JS